CRM/Core/OptionGroup.php

```diff
@@ -200,8 +200,8 @@ protected static function flushValues($name, $flip, $grouping, $localize, $condi
   /**
    * @return string
    */
-  protected static function createCacheKey() {
-    $cacheKey = "CRM_OG_" . serialize(func_get_args());
+  protected static function createCacheKey($id) {
+    $cacheKey = "CRM_OG_" . preg_replace('/[^a-zA-Z0-9]/', '', $id) . '_' . md5(serialize(func_get_args()));
```

Comment:
why not use cleanKey here?

Reply:
Could do it -- it'll work. The reason I didn't was cosmetic...

Anecdotally, `cleanKey(...serialize(func_get_args())...)` degraded to md5 form for every example I observed. (Which makes sense -- there's a lot of inputs, and `serialize` format uses a lot of meta chars.) This formula provides an intelligible prefix.